.storybook/main.ts

```diff
@@ -49,7 +44,6 @@ const config: StorybookConfig = {
     // Type-check stories during Storybook build.
     check: true,
     // Display the compiled value options.
-    reactDocgen: "react-docgen-typescript",
```

Comment:
The default is "react-docgen" in S9 and is recommended. We don't lose any of the props in the args tables.